tests/foreman/cli/test_subscription.py

```diff
@@ -284,3 +284,29 @@ def test_positive_auto_attach_disabled_golden_ticket(
     with pytest.raises(CLIReturnCodeError) as context:
         target_sat.cli.Host.subscription_auto_attach({'host-id': host_id})
     assert "This host's organization is in Simple Content Access mode" in str(context.value)
+
+
+@pytest.mark.tier2
+def test_positive_prepare_for_sca_only_hammer(function_org, target_sat):
+    """Verify that upon certain actions, Hammer notifies users that Simple Content Access
+        will be required for all organizations in the next release
+
+    :id: d9985a84-883f-46e7-8352-dbb849dbfa34
+
+    :expectedresults: Hammer returns a message notifying users that Simple Content Access
+        will be required for all organizations in the next release
+    """
+    org_results = target_sat.execute(
+        f'hammer organization update --id {function_org.id} --simple-content-access false'
+    )
+    assert (
+        'Simple Content Access will be required for all organizations in the next release'
+        in str(org_results.stderr)
+    )
+    sca_results = target_sat.cli.SimpleContentAccess.disable(
+        {'organization-id': function_org.id}, raw=True
+    )
+    assert (
+        'Simple Content Access will be required for all organizations in the next release'
+        in str(sca_results.stderr)
```

Comment:
```suggestion
        in sca_results.stderr
```

Reply:
This does not work. I turned it into a str so i can assert on it with text